### ISSUE TITLE
album and artist is undefined

### ISSUE THREAD

**Issue by dimactavishy** [zyrouge/genius-lyrics#23]
So i was making a lyrics command on my Discord bot using this package, and everytime i searched a song it will always return this:
Artist: [object Object]
Album: undefined
![anu](https://user-images.githubusercontent.com/79780581/145402202-35329216-6273-45e5-a698-19271bbf31e5.PNG)
Lyrics are fine though, as you can see.

And here's my code:
```
const Genius = require("genius-lyrics");
const Gclient = new Genius.Client
const { MessageEmbed } = require("discord.js");

module.exports = {
    info: {
        name: "lyrics",
        description: "Will return the lyrics of a specified song.",
        usage: "[song name]",
        cooldown: 10,
    },

    execute: async function (client, message, args, Discord) {

        var searchString = args.join(" ");
        if (!searchString) return message.reply("please specify a keyword, i am not able to read your thoughts.").then(msg => { msg.delete({ timeout: 5000 }) });

        const searches = await Gclient.songs.search(searchString);
        const firstSong = searches[0];
        const lyrics = await firstSong.lyrics();

        let lyricEmbed = new MessageEmbed()
            .setColor('BLUE')
            .setTitle(firstSong.title)
            .setURL(firstSong.url)
            .setThumbnail(firstSong.thumbnail)
            .setDescription(
                `Artist: ${firstSong.artist}\n`
                + `Album: ${firstSong.album}\n\n`
                + `${lyrics}`
            )
            .setTimestamp();
        message.channel.send(lyricEmbed);

    }
}
```
I'm an amateur, so i probably screwed up the code a little bit.

**Comment by zyrouge** [comment]
> So i was making a lyrics command on my Discord bot using this package, and everytime i searched a song it will always return this: Artist: [object Object] Album: undefined ![anu](https://user-images.githubusercontent.com/79780581/145402202-35329216-6273-45e5-a698-19271bbf31e5.PNG) Lyrics are fine though, as you can see.
> 
> And here's my code:
> 
> ```
> const Genius = require("genius-lyrics");
> const Gclient = new Genius.Client
> const { MessageEmbed } = require("discord.js");
> 
> module.exports = {
>     info: {
>         name: "lyrics",
>         description: "Will return the lyrics of a specified song.",
>         usage: "[song name]",
>         cooldown: 10,
>     },
> 
>     execute: async function (client, message, args, Discord) {
> 
>         var searchString = args.join(" ");
>         if (!searchString) return message.reply("please specify a keyword, i am not able to read your thoughts.").then(msg => { msg.delete({ timeout: 5000 }) });
> 
>         const searches = await Gclient.songs.search(searchString);
>         const firstSong = searches[0];
>         const lyrics = await firstSong.lyrics();
> 
>         let lyricEmbed = new MessageEmbed()
>             .setColor('BLUE')
>             .setTitle(firstSong.title)
>             .setURL(firstSong.url)
>             .setThumbnail(firstSong.thumbnail)
>             .setDescription(
>                 `Artist: ${firstSong.artist}\n`
>                 + `Album: ${firstSong.album}\n\n`
>                 + `${lyrics}`
>             )
>             .setTimestamp();
>         message.channel.send(lyricEmbed);
> 
>     }
> }
> ```
> 
> I'm an amateur, so i probably screwed up the code a little bit.

The `artist` property is an object, and `[object Object]` is caused by stringifying it. I'll have to check about the `album` property.

**Comment by dimactavishy** [comment]
> The artist property is an object, and [object Object] is caused by stringifying it. Most of the properties will be undefined when using search as it doesn't provide all the info. If you need all the info, you can use firstSong.fetch() which will get it's complete details. I'm closing this since it's not an issue but feel free to reopen this if needed.

then what about `album`?

**Comment by dimactavishy** [comment]
> The artist property is an object, and [object Object] is caused by stringifying it. Most of the properties will be undefined when using search as it doesn't provide all the info. If you need all the info, you can use firstSong.fetch() which will get it's complete details. I'm closing this since it's not an issue but feel free to reopen this if needed.

i tried doing that with `const songInfo = await firstSong.fetch();` and it returns this:

```
(node:4) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'album' of undefined
at Song.<anonymous> (/app/node_modules/genius-lyrics/dist/Songs/Song.js:89:38)
at Generator.next (<anonymous>)
at fulfilled (/app/node_modules/genius-lyrics/dist/Songs/Song.js:5:58)
at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

**Comment by zyrouge** [comment]
Has been fixed in v4.3.5! 🎉
